packages/cycle-scripts/package.json

```diff
@@ -30,12 +30,16 @@
     "cycle-scripts": "./index.js"
   },
   "dependencies": {
-    "babel-core": "^6.24.1",
```

Comment:
Yes, but if something cause an issue with a specific issue we cannot lock for example babel-core... anyway. Good job. I'll review the rest later